packages/expect/package.json

```diff
@@ -36,11 +36,10 @@
     "@types/chai": "catalog:",
     "@vitest/spy": "workspace:*",
     "@vitest/utils": "workspace:*",
-    "chai": "catalog:",
     "tinyrainbow": "catalog:"
   },
   "devDependencies": {
     "@vitest/runner": "workspace:*",
-    "rollup-plugin-copy": "^3.5.0"
+    "chai": "catalog:"
```

Comment:
Latest `^5` and `6` versions should be good now, no need for bundling.

```js
const startTime = performance.now();

await import("chai");
// "chai": "5.2.1" > 4.39 ms
// "chai": "5.3.1" > 13.52 ms
// "chai": "5.3.2" > 2.92 ms
// "chai": "6.0.0" > 2.87 ms

console.log(`${(performance.now() - startTime).toFixed(2)} ms`);
```

Reply:
Sounds good. Let's keep `chai` in dependencies.